### DISCUSSION TITLE
How does the documentation shared in the readme get updated?

### DISCUSSION THREAD

**Discussion by SoulPancake** [redis/go-redis#2493] [Q&A]
As we have recently built support for some more commands, But I don't see them updated on the docs
So, Where can we access that and how can we contribute there as well?
@vmihailenco might be able to give some ideas on this

**Comment by monkey92t** [comment]
https://github.com/go-redis/redis-docs   Is this what you want?

**Reply by SoulPancake** [reply]
@monkey92t Is this the same thing deployed https://pkg.go.dev/github.com/redis/go-redis/v9 < here ?

**Reply by monkey92t** [reply]
> @monkey92t Is this the same thing deployed https://pkg.go.dev/github.com/redis/go-redis/v9 < here ?

I understand what you mean, we need to release a new version, currently v9.0.2 :)
